README.md

```diff
@@ -377,6 +386,7 @@ run against the same database (via GM_DBURL envvar) unless --shadow or
 
 Options:
   --help          Show help                                            [boolean]
+  --config Optional path to .gmrc(.js) - default is `${pwd}.gmrc`      [string]
```

Comment:
It looks like you updated all of these manually (because the alignment is off); instead, you can use the `scripts/update-docs.js` script.